lib/posix-user/exportsyms.uk

```diff
@@ -64,3 +64,6 @@ getgrnam_r
 setgrent
 endgrent
 getgrent
+getresuid
```

Comment:
Just a nitpick. This is redefined ([line 50](https://github.com/unikraft/unikraft/blob/7ac68b174b5625b41262a1dd8076af02a4b4e4a6/lib/posix-user/exportsyms.uk#L49)).